zerver/lib/bugdown/__init__.py

```diff
@@ -55,6 +54,14 @@
     # mypy requires the Optional to be inside Union
     ElementStringNone = Union[Element, Optional[text_type]]
 
+def unescape(s):
+    # type: (text_type) -> (text_type)
+    if six.PY2:
+        return html_parser.HTMLParser().unescape(s)
+    else:
+        import html
```

Comment:
maybe would be cleaner to do an `if six.PY3: import html` at the top so that we don't have imports in the middle of the file?